cli/tsc/99_main_compiler.js

```diff
@@ -303,6 +334,49 @@ delete Object.prototype.__proto__;
     return isNodeSourceFile;
   });
 
+  /**
+   * @param msg {string}
+   * @param code {number}
+   */
+  function formatMessage(msg, code) {
```

Comment:
Moved this from Rust to TypeScript so that it's always deserialized the same way at the source in both the lsp and for `deno check`.